hoot-core-test/src/test/cpp/hoot/core/io/OsmPbfWriterTest.cpp

```diff
@@ -128,18 +128,19 @@ class OsmPbfWriterTest : public HootTestFixture
     r->setTag("note", "test tag");
     writer.writePb(r, &ss);
 
-    //_dumpString(ss.str());
+    //TestUtils::dumpString(ss.str());
```

Comment:
Changing this test output was necessary due to adding relation type as a tag to relation pbf output.